maturity-level-one/src/Controllers/PlayersController.cs

```diff
@@ -22,7 +22,7 @@ namespace maturity_level_one.Controllers
     [ValidateModel]
     public class PlayersController : ControllerBase
```

Comment:
For controller methods, I would not suffix them as they also define the OperationId and would be odd for a consumer to reflect implementation detail. Thoughts @MassimoC?

All other methods I personally suffix with `Async` as well but for some people this is a sensitive topic.

See [Defining OperationIds
](https://github.com/CoditEU/practical-api-guidelines/tree/master/maturity-level-one#defining-operationids)

Reply:
OK, I can agree with not suffixing controller methods.   I'll revert that

Reply:
Cool, thanks!